documentation/service-rules/ecf1/appropriate-body-users.md

```diff
@@ -393,7 +393,7 @@ expand to other academic years.
 - TRN
 - School URN
 - Status (defined below)
-- Induction type (FIP, CIP or DIY)
+- Induction type (Provider led or School led)
```

Comment:
in ecf1 let's make a note around the value in the db are old, but aesthetically we show the new ones